Wikipedia/Code/wikipedia-languages.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "languageCode" : "ab",
-    "languageName" : "Аҧсшәа",
+    "languageName" : "аԥсшәа",
```

Comment:
This languages json is occasionally generated manually from our script at WikipediaLanguagesCommandLineUtility.swift. This script internally references the site info endpoint [here](https://meta.wikimedia.org/w/api.php?action=sitematrix&smsiteprop=url%7Cdbname%7Ccode%7Csitename%7Clang&format=json&formatversion=2&origin=*), which seems to return the old spelling, so your change here will get overwritten the next time we run this. 

We have already added some workarounds for this endpoint in this script, so you could add something like this [here](https://github.com/amire80/wikipedia-ios/blob/b9082479eeb63398d295352c8ddaaba380861080/Command%20Line%20Tools/Update%20Languages/WikipediaLanguageCommandLineUtilityAPI.swift#L54) to retain this spelling. Of course ideally the site info endpoint will get updated, but this is just to give you a client-side fix option.

```
guard code != "ab" else {
    return Wikipedia(languageCode: code, languageName: "аԥсшәа", localName: localname, altISOCode: nil)
}
```

Reply:
Thanks for the comment! The name was updated in core, and that change was deployed, so the new name is seen in the code. Can you perhaps run that updating now? That letter also still appears in Wikipedia/assets/codemirror/config/codemirror-config-ab.json as part of the old linkTrail, which was also updated in core; do you update that automatically as well?

Reply:
@amire80 thanks! It's looking good, I ran the script locally and didn't see this overwritten in `wikipedia-languages.json`, so we're good there. We do generate the `codemirror-config-ab.json` file via the same script, unfortunately running it locally didn't seem to make any changes. I did some digging and it looks like it's pulling that data from [here](https://ab.wikipedia.org/w/load.php?debug=false&lang=en&modules=ext.CodeMirror.data), which is still returning the old character. If this needs to be fixed, we should also fix it at that endpoint.